<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ARMIDEsigner</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QTabWidget" name="tabWidget" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>651</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab" >
     <attribute name="title" >
      <string>Start</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>520</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_New" />
    <addaction name="action_Open" />
    <addaction name="separator" />
    <addaction name="action_Save" />
    <addaction name="separator" />
    <addaction name="action_Exit" />
   </widget>
   <addaction name="menuFile" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="actionUpload" />
   <addaction name="actionBuild" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset>
     <normaloff>images/black-folder-new.png</normaloff>images/black-folder-new.png</iconset>
   </property>
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset>
     <normaloff>images/black-document-open.png</normaloff>images/black-document-open.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset>
     <normaloff>images/document-save.png</normaloff>images/document-save.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="action_New" >
   <property name="text" >
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save" >
   <property name="text" >
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionUpload" >
   <property name="icon" >
    <iconset>
     <normaloff>images/go-top.png</normaloff>images/go-top.png</iconset>
   </property>
   <property name="text" >
    <string>Upload</string>
   </property>
  </action>
  <action name="actionBuild" >
   <property name="icon" >
    <iconset>
     <normaloff>images/gkdebconf.png</normaloff>images/gkdebconf.png</iconset>
   </property>
   <property name="text" >
    <string>Build</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
